client/Commands/CombatantCommander.ts

```diff
@@ -145,8 +145,10 @@ module ImprovedInitiative {
                 return false;
             }
 
-            const combatantNames = suggestedCombatants.map(c => c.Name()).join(', ');
-            const prompt = new AcceptDamagePrompt(suggestedCombatants, suggestedDamage, suggester, this.tracker.EventLog.LogHPChange);

```

Comment:
Have this parameter just take the whole tracker object so this behavior can be moved into the AcceptDamagePrompt class.